docs/snippets/logger/basicUsage.ts

```diff
@@ -3,5 +3,5 @@ import { Logger } from '@aws-lambda-powertools/logger';
 const logger = new Logger({ serviceName: 'serverlessAirline' });
 
 export const handler = async (_event, _context): Promise<void> => {
-    // ...
+  // ...
```

Comment:
Let's add a log, i.e. `logger.info('Hello World');`, and remove the empty comment line (`// ...`)